src/components/markdown/markdown.test.tsx

```diff
@@ -55,6 +55,19 @@ describe('<MarkdownSlide />', () => {
     expect(wrapper.find('li').at(1).children().find('i')).toHaveLength(2);
     expect(wrapper.find('li').at(2).children()).toHaveLength(1);
   });
+
+  it('should generate line breaks for inline paragraph elements', () => {
+    const wrapper = mountInsideDeck(
+      <MarkdownSlide>{`
+        One
+        **Two**
+        _Three_
+        \`Four\`
+        ~~Five~~
+      `}</MarkdownSlide>
+    );
+    expect(wrapper.find('br')).toHaveLength(4);
+  });
```

Comment:
Could we add some win + mixed tests here? Thinking something like:

```suggestion

  it('should generate line breaks for inline paragraph elements with newline', () => {
    const wrapper = mountInsideDeck(
      <MarkdownSlide>{`
        One
        **Two**
        _Three_
        \`Four\`
        ~~Five~~
      `}</MarkdownSlide>
    );
    expect(wrapper.find('br')).toHaveLength(4);
  });

  it('should generate line breaks for inline paragraph elements with CR', () => {
    const wrapper = mountInsideDeck(
      <MarkdownSlide>{`One\r\n**Two**\r\n_Three_\r\n\`Four\`\r\n~~Five~~`}</MarkdownSlide>
    );
    expect(wrapper.find('br')).toHaveLength(4);
  });

  it('should generate line breaks for inline paragraph elements with mixed returns', () => {
    const wrapper = mountInsideDeck(
      <MarkdownSlide>{`One\n**Two**\r\n_Three_\n\`Four\`\r\n~~Five~~`}</MarkdownSlide>
    );
    expect(wrapper.find('br')).toHaveLength(4);
  });
```

Reply:
My bad 🤦‍♂️. I totally missed the part where you asked me to make tests for these. I thought you meant a regex test.

Reply:
I just added these tests.